pkg/graph/graphs.go

```diff
@@ -134,9 +134,10 @@ func DeleteRecursive(ctx context.Context, db driver.Database, id driver.Document
 const getWithAccessLevel = `
 FOR path IN OUTBOUND K_SHORTEST_PATHS @account TO @node
 GRAPH @permissions SORT path.edges[0].level
+    LET ns = (FOR ns IN Namespaces FILTER ns._id == path.edges[0]._to RETURN ns)[0]
```

Comment:
1. Node's Namespace is the last Node before the Target Node, not the first (path.edges[-1])
2. Namespaces would always(unless nested instance) be simply a previous node in the path, so path.vertices[-2]
3. If 2 is not the case, then entity just won't have namespace in the access field
4. Example [here](https://github.com/infinimesh/infinimesh/blob/05c650fe3349febd3139766f8b7a8535a8903f81/pkg/graph/common.go#L95-L100)